localisation/00_zuwc_united_localisation.csv

```diff
@@ -2969,6 +2969,7 @@ EVTDESC_WCPRM_502;As you was ready and done every preparation, Candle Ritual beg
 EVTDESC_WCPRM_503;As you was ready and done every preparation, Candle Ritual began in dark place. As you lit every candle, darkness fades away from you. When you burnt all ingredients, you began to look into fire of candles. You see something that can't be explained, but still you have a feeling that this was a good omen.;;Als du bereit und Vorbereitungen getroffen, Kerzenritual begann an dunklem Ort. Als du Kerzen angez�ndet, Dunkelheit verblasste vor dir. Als du alle Zutaten verbrannt, du schauen in Kerzenfeuer. Du etwas sehen, das nicht erkl�rt, aber du trotzdem haben Gef�hl, dass dies gutes Omen.;;;;;;;;;;;x
 EVTOPTA_WCPRM_503;Candle protects me.;;Kerze mich sch�tzen.;;;;;;;;;;;x
 EVTOPTB_WCPRM_500;No! This too expensive!;;Nein! Das zu teuer!;;;;;;;;;;;x
+old_gods_worship_FEATURES;�Y*�! Can chose their main Old God in the decision menu.\n�Y*�! Can use offmap screen of chosen Old God.\n�Y*�! Can recieve offmap currency of chosen Old God by sacrificing characters and sieging holdings.;;;;;;;;;;;;;x
```

Comment:
why the  §Y*§!  ? the star won't appear yellow in the features window, right?

Reply:
Huh?

![image](https://user-images.githubusercontent.com/46576860/84263624-7f3d8080-ab28-11ea-895c-162fd000002e.png)

Reply:
ah, my bad, i checked in the first info-window only 
![grafik](https://user-images.githubusercontent.com/47716948/84264061-d63f5780-ab20-11ea-8a55-ba142918bd42.png)